etc/notebooks/examples/urth-viz-table.ipynb

```diff
@@ -16,15 +16,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<link rel=\"stylesheet\" type=\"text/css\" href=\"bcard.css\">\n",
+       "<link rel=\"import\" href=\"urth_components/urth-help/urth-help.html\" is=\"urth-core-import\">"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
     "%%html\n",
     "<link rel=\"stylesheet\" type=\"text/css\" href=\"bcard.css\">\n",
-    "<link rel=\"import\" href=\"urth_components/urth-viz-table/urth-viz-table.html\" is=\"urth-core-import\">"
+    "<link rel=\"import\" href=\"urth_components/urth-help/urth-help.html\" is=\"urth-core-import\">"
```

Comment:
@aluu317 I think this is wrong. Why are you using `urth-help` in the table example?